### DISCUSSION TITLE
Click on include link to open it

### DISCUSSION THREAD

**Discussion by xtimbeau** [quarto-dev/quarto-cli#11143] [Feature Requests]
### Description

It would be nice if by clicking on `{{< include path/to_a_qmd.qmd >}}` that could open the windows in RStudio or VS code or Positron. Could be limited to visual mode.

**Comment by mcanouil** [comment]
That's not something Quarto CLI can do.
Quarto CLI has no interface (outside a terminal).
That's a request to make on the IDE's repositories:

- https://github.com/rstudio/rstudio
- https://github.com/posit-dev/positron


Note that VSCode (so should Positron) does recognise file paths but not when escaped inside `{{ }}`.

The Visual Editor is a separate tool integrated by the mentioned IDE or extensions:

- https://github.com/quarto-dev/quarto
